cranelift/codegen/src/egraph/elaborate.rs

```diff
@@ -621,6 +632,21 @@ impl<'a> Elaborator<'a> {
                 self.func.dfg.inst_args_mut(inst)[i] = arg.value;
             }
 
+            if let Some(mut dest) = self.func.dfg.insts[inst].branch_destination() {
+                for i in 0..dest.args_slice(&self.func.dfg.value_lists).len() {
```

Comment:
I've added some functions to abstract over the values of an instruction to avoid this sort of duplication, and am in the process of auditing uses of the existing arg accessors to determine if they need to be changed.